package.json

```diff
@@ -42,14 +42,15 @@
 		"lint:format": "prettier --cache --check .",
 		"lint": "concurrently --prefix \"lint:{name}\" --pad-prefix --group \"node:lint:*\"",
 		"types:root": "tsc -p tsconfig.json",
+		"types:shared": "tsc -p src/shared/tsconfig.json",
 		"types:main": "tsc -p src/main/tsconfig.json",
 		"types:services": "tsc -p src/services/tsconfig.json",
 		"types:preload": "tsc -p src/preload/tsconfig.json",
 		"types:renderer": "tsc -p src/renderer/tsconfig.json",
 		"types": "concurrently --prefix \"types:{name}\" --pad-prefix --group \"node:types:*\"",
-		"test": "concurrently --pad-prefix --group \"node:types\" \"node:vitest:run\"",
-		"vitest:run": "vitest run --config ./src/renderer/vite.config.js",
-		"vitest:watch": "vitest watch --config ./src/renderer/vite.config.js",
+		"test:renderer": "vitest run --config ./src/renderer/vite.config.js",
+		"test:shared": "vitest run --dir ./src/shared/",
+		"test": "concurrently --prefix \"test:{name}\" --pad-prefix --group \"node:test:*\"",
```

Comment:
Eventually should be using [Vitest projects](https://vitest.dev/guide/projects.html) as opposed to separate commands (I think). Something to address later on!